en/django_start_project/README.md

```diff
@@ -61,6 +62,14 @@ TIME_ZONE = 'Europe/Berlin'
 Modifying "Europe/Berlin" as appropriate
 
 
+We'll also need to add a path for static files (we'll find out all about static files and CSS later in the tutorial).  Go down to  the *end* of the file, and just underneath  the `STATIC_URL` entry, add a new one called `STATIC_ROOT`:
+
+```python
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static') 
```

Comment:
Can we use `os.path.join(BASE_DIR, 'collected_static')` just to make sure there's no confusion?